.azure-pipelines/steps/run.yml

```diff
@@ -35,11 +35,14 @@ steps:
   displayName: Install build dependencies (OSX)
   condition: and(succeeded(), eq(variables['Agent.OS'], 'Darwin'), eq(variables['RUST_CHECK_TARGET'],'dist'))
 
+# Switch to XCode 9.3 on OSX since it seems to be the last version that supports
+# i686-apple-darwin. We'll eventually want to upgrade this and it will probably
+# force us to drop i686-apple-darwin, but let's keep the wheels turning for now.
```

Comment:
The last 32-bit macs were from mid 2011, so we are probably ok dropping them...

Reply:
Well, it's still a Tier 1 platform, so we can't just drop support in this PR.